app/components/post_profile_picture/post_profile_picture.js

```diff
@@ -12,29 +12,47 @@ import webhookIcon from 'assets/images/icons/webhook.jpg';
 
 const PROFILE_PICTURE_SIZE = 32;
 
-function PostProfilePicture(props) {
-    const {
-        enablePostIconOverride,
-        fromWebHook,
-        isSystemMessage,
-        onViewUserProfile,
-        overrideIconUrl,
-        theme,
-        userId
-    } = props;
-    if (isSystemMessage) {
-        return (
-            <View>
-                <AppIcon
-                    color={theme.centerChannelColor}
-                    height={PROFILE_PICTURE_SIZE}
-                    width={PROFILE_PICTURE_SIZE}
-                />
-            </View>
-        );
-    } else if (fromWebHook) {
-        if (enablePostIconOverride) {
+export default class PostProfilePicture extends React.PureComponent {
```

Comment:
Minor nit, would change the import to `import React, {PureComponent} from 'react'` and extend `PureComponent` to match the components.

Reply:
This is what I get for switching between the web app and RN. I'm mixing up the different styles :p